src/app/project/controllers/collect-records.ctrl.js

```diff
@@ -57,12 +59,15 @@ angular.module('app.project').controller('CollectRecordsCtrl', [
       ProjectService.HABITAT_COMPLEXITY_TRANSECT_TYPE,
       ProjectService.BLEACHING_QC_QUADRAT_TYPE
     ];
-
     var statusChoices = [null, 'ok', 'warning', 'error'];
+    var protocolMethodsLength = protocolMethods.length;
```

Comment:
Not seeing the need for additional vars; these are only used in one place each. All comments on this file apply also to transect methods below.